enterprise-suite/alertmanager/alertmanager.yml

```diff
@@ -13,7 +13,7 @@ route:
   # Wait before sending a new alert for a group.
   group_interval: 5m
   # Group alerts by namespace and workload.
```

Comment:
Might want to update the comment too, although it seems kinda redundant anyway.

Reply:
Indeed! Don't want to burden a PR just for this, so I'll fix it next time.